core/src/main/scala/org/apache/spark/storage/ExternalBlockStore.scala

```diff
@@ -40,7 +40,7 @@ private[spark] class ExternalBlockStore(blockManager: BlockManager, executorId:
       externalBlockManager.map(_.getSize(blockId)).getOrElse(0)
     } catch {
       case NonFatal(t) =>
-        logError(s"error in getSize from $blockId", t)
+        logError(s"Error in getSize from $blockId", t)
```

Comment:
and do this for the other messages too, since it is not clear "getSize", "getValues", ... are function calls. Let's just make them more explicitly.